src/iexamine.cpp

```diff
@@ -1841,11 +1841,13 @@ void iexamine::aggie_plant( player &p, const tripoint &examp )
         } else {
             std::list<item> harvest_items = get_harvest_items( seed );
             proceed_plant_after_harvest( examp );
```

Comment:
It is no longer safe to access seed variable after proceed_plant_after_harvest is called, because that function calls `g->m.i_clear( examp );` which kills the seed you are referring to. If you need some information from seed, you should store it in some temporary variable before calling the proceed_plant_after_harvest.